contracts/upgrader/tests/atomic_upgrades.rs

```diff
@@ -23,22 +24,21 @@ fn upgrade_and_migrate_are_atomic() {
     let original_version: String = dummy_client.version();
     assert_eq!(original_version, String::from_str(&env, "0.1.0"));
 
-    let (upgrade_auth, migrate_auth) =
-        build_invocation_auths(&env, &contract_address, &hash_after_upgrade, &expected_data);
-
-    // add the owner to the set of authenticated addresses
-    env.mock_auths(&[
-        MockAuth {
-            address: &owner,
-            invoke: &upgrade_auth,
-        },
-        MockAuth {
-            address: &owner,
-            invoke: &migrate_auth,
-        },
-    ]);
+    let upgrader = UpgraderClient::new(&env, &upgrader_address);
 
-    UpgraderClient::new(&env, &upgrader_address).upgrade(
+    let upgrade_auth = mock_auth! {
+        env,
+        owner,
+        dummy_client.upgrade(hash_after_upgrade)
+    };
+
+    let migrate_auth = mock_auth! {
+        env,
+        owner,
+        dummy_client.migrate(expected_data)
+    };
```

Comment:
```suggestion
    let upgrade_auth = mock_auth!(
        env,
        owner,
        dummy_client.upgrade(hash_after_upgrade)
    );
    let migrate_auth = mock_auth!(
        env,
        owner,
        dummy_client.migrate(expected_data)
    );
```
use parens for consistency